src/components/MaterialFilter.tsx

```diff
@@ -137,27 +143,67 @@ const MaterialFilter: React.FC = () => {
               </label>
             </>
           ))}
-        {/* <label>
+        <label>
           Weight
-          <input className="filter-select" placeholder="Weight" id="weight"></input>
-        </label> */}
+          <input
+            type="number"
+            className="filter-select"
+            id="weight"
+            placeholder="Weight"
+            required
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            onChange={(e) => setSelectedWeight(e.target.value as any)}
+          ></input>
+          <select
+            className="filter-select"
+            id="unit"
+            onChange={(e) => setSelectedWeightUnit(e.target.value as string)}
+          >
+            <option>lb</option>
+            <option>kg</option>
+          </select>
+        </label>
+        <label>
+          Material Source
+          {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
+          <select
+            className="filter-select"
+            id="material-source"
+            onChange={(e) => setSelectedMaterialSource(e.target.value as string | null)}
+            required
+          >
+            <option value=""></option>
+            <option value="Post Consumer">Post Consumer</option>
+            <option value="Post Industrial">Post Industrial</option>
+          </select>
+        </label>
         <label>
-          <label>
-            Material Source
-            {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
-            <select onChange={(e) => setSelectedMaterialSource(e.target.value as any)}>
-              <option placeholder="Select">Select one</option>
-              <option value="Post Consumer">Post Consumer</option>
-              <option value="Post Industrial">Post Industrial</option>
-            </select>
-          </label>
-          {/* I am a...
-          <select>
+          Material Postal Code Location
+          <input
+            type="text"
+            className="filter-select"
+            id="zip"
+            placeholder="Zip Code"
```

Comment:
And one more placeholder text to remove.

Reply:
Removed `placeholder="Zip Code"`